CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Mapbox welcomes participation and contributions from everyone.
   ```
 - Added guarantees that route progress with `RouteProgress#currentState == OFF_ROUTE` arrives earlier than `NavigationRerouteController#reroute` is called. [#6764](https://github.com/mapbox/mapbox-navigation-android/pull/6764)
 - Fixed a rare `java.lang.NullPointerException: Attempt to read from field 'SpeechAnnouncement PlayCallback.announcement' on a null object reference` crash in `PlayCallback.getAnnouncement`. [#6760](https://github.com/mapbox/mapbox-navigation-android/pull/6760)
+- Changed distance formatting: now all the imperial distances down to 0.1 miles will be represented in miles. [#6775](https://github.com/mapbox/mapbox-navigation-android/pull/6775)
```

Comment:
```suggestion
- Changed distance formatting: now all the imperial distances down to `0.1` miles will be represented in miles, otherwise feet. [#6775](https://github.com/mapbox/mapbox-navigation-android/pull/6775)
```